spec/features/user_can_view_items_in_cart_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+describe 'authorized user can add items to cart', type: :feature do
+  let(:item1) { item = Fabricate.build(:item)
+  item.categories << category1
+  item.save
+  item}
+  let(:item2) { item = Fabricate.build(:item)
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.